### ISSUE TITLE
Replace list of report reasons on user profiles

### ISSUE THREAD

**Issue by 0x84f** [ppy/osu-web#11494]
A GMT brought up that the current report reasons listed on the profile are a bit redundant and have a greater chance of users submitting reports that go through to invalid channels, after a Discord discussion in osu!internal [this new listing](https://discord.com/channels/90072389919997952/276296389519081472/1284144810697162803) was agreed upon between the team.

For ease of access I'll include the proposed list here as well:

- Cheating
- Using multiple accounts
- Inappropriate chat behaviour
- Inappropriate user profile
- Other

| current | proposed |
|----------|----------|
| ![b4](https://github.com/user-attachments/assets/6d0232bf-7041-49c8-b9ea-4fe388a97cac) | ![after](https://github.com/user-attachments/assets/ca6eb761-5f4f-4731-be67-69492592e8d6)|

Basically opening a GitHub issue so it's easier to track dev/peppy discussions around changing this.

**Comment by peppy** [comment]
This is fine to go ahead with IMO.

**Comment by notbakaneko** [comment]
Do we want to preserve the existing enums (and point them to new meanings) or make new ones?

**Comment by peppy** [comment]
Either or is fine, IMO. Whatever requires the least migration effort.

**Comment by 0x84f** [comment]
bumping this

**Comment by nanaya** [comment]
is "Inappropriate user profile" different from "(Linking to) inappropriate content"? (also used for beatmap and other things)

**Comment by peppy** [comment]
> is "Inappropriate user profile" different from "(Linking to) inappropriate content"? 

Should be the same thing, context being the URL.

> Also the other types are still being used somewhere else so I don't know if migration is even necessary

Fair
